tests/end-to-end/backup-restore-test/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.9-alpine3.7 as builder
```

Comment:
Please use `alpine` image here. As used here: https://github.com/kyma-project/kyma/blob/master/tests/kubeless-integration/Dockerfile

Reply:
Updated to the latest alpine based golang image. This image is already used across kyma multiple times.